ramls/examples/instance_post.json

```diff
@@ -18,5 +18,33 @@
     "tagList" : [
       "important"
     ]
-  }
+  },
+  "precedingTitles": [
+    {
+      "id": "cb4f8537-7643-4ef4-8cc4-6dbf7bbe8a6b",
+      "precedingInstanceId": "1b74ab75-9f41-4837-8662-a1d99118008d",
+      "title": "A semantic web primer",
+      "hrid": "inst000000000022",
+      "identifiers": [
+        {
+          "identifierTypeId": "8261054f-be78-422d-bd51-4ed9f33c3422",
+          "value": "0262012103"
+        }
+      ]
```

Comment:
Is all of this information needed in the post to create a new instance?

This appears to be a connected preceding title, does that need this information at all?

Reply:
we don't need all fields for a connected preceding title. Need only `precedingInstanceId` and `succeedingInstanceId`. It is just an example with all available fields